src/applications/personalization/profile/tests/e2e/direct-deposit/update-flow.cypress.spec.js

```diff
@@ -71,7 +71,9 @@ describe('Direct Deposit', () => {
   describe('for CNP', () => {
     it('should allow bank info updates, show WIP warning modals, show "update successful" banners, etc.', () => {
       cy.axeCheck();
-      cy.findByRole('button', { name: /add.*bank info/i }).click({
+      cy.findByLabelText(
```

Comment:
Could you keep this as a `findByRole` selector? [RTL suggests using `byRole()` selectors when possible](https://testing-library.com/docs/queries/about/#priority). It helps ensure the page is accessible to assistive tech.

Reply:
I think I can. Not familiar with these commands so I will have to figure the best way to manipulate that one.